modules/subscription/providers.tf

```diff
@@ -0,0 +1,4 @@
+provider "google" {
```

Comment:
We didn't want to have a provider hardcoded here because you could use old version with old provider with a subset of fields... you can see that in the readme.

I would be ok of dropping support for this with a switch to tf12 but we need to change also the readme. I would remove a note which is there and write the min. version of google provider in compatibility section

Reply:
👍